### PR TITLE
Update advanced-configuration.html

### PR DESCRIPTION
**Pull request by kingwarrick**
to include information about reverse proxy.

### CODE REVIEW

**Comment by kingwarrick** [comment]
Updated with additional instructions to trigger reload

File: advanced-configuration.html [hunk 1]
```diff
@@ -115,6 +115,17 @@ <h2 id="filtering">Sieve Filtering</h2>
 				  <p>Note that Mail-in-a-Box contains a sieve script to filter Spam which cannot be altered. When the header contains X-Spam-Status with value Yes, the email is filed into the Spam mailbox.</p>
 
 				  <p>For information on what is possible with Dovecot Pigeonhole Sieve, as well as examples, see <a href="http://wiki2.dovecot.org/Pigeonhole/Sieve">http://wiki2.dovecot.org/Pigeonhole/Sieve</a></p>
+					
+					<h2>MiaB as Reverse Proxy</h2>
```
Comment (JoshData) [inline advanced-configuration.html:119]: The "MiaB" abbreviation isn't used in our documentation. Spell it out as "Mail-in-a-Box".

File: advanced-configuration.html [hunk 2]
```diff
@@ -115,6 +115,17 @@ <h2 id="filtering">Sieve Filtering</h2>
...
+					<p>If you are running multiple services on the same IP address as your Mail-in-a-Box server, you can use nginx within MiaB to proxy to those services. By editing <code>/home/user-data/www/custom.yaml</code>, proxy and/or redirect can be achieved:</p>
```
Comment (JoshData) [inline advanced-configuration.html:120]: Same here.

Comment (JoshData) [inline advanced-configuration.html:120]: This isn't limited to services on the same IP address.

File: advanced-configuration.html [hunk 3]
```diff
@@ -115,6 +115,17 @@ <h2 id="filtering">Sieve Filtering</h2>
...
+					<p><code><pre>
+					my-domain.com:
+					   redirects:
+						   ^/foo: https://example.com/bar
+						 proxies:
+						   /baz: http://internal.ip:9090 </pre></code></p>
+					<p>The configuration must be regenerated/reloaded before this takes effect. You can do so by navigating to Web within the admin panel and clicking on any of the Change buttons or simply run <code>sudo ~/mailinabox/tools/web_update</code>.</p>
+					<p>See https://github.com/mail-in-a-box/mailinabox/issues/608#issuecomment-279204357 for more discussion and use cases</p>
```
Comment (JoshData) [inline advanced-configuration.html:128]: The URL should be a HTML link. There should be a period at the end of the sentence.

**Review by JoshData — CHANGES_REQUESTED** [review]

**Pull request closed without merging**